scalableminds/webknossos
# In dataset upload, catch AccessDenied exception for better error reporting

@daniel-wer You will still not be able to upload datasets on dev instances, but at least you will know why ;-)

### URL of deployed dev instance (used for testing):
- https://catchdatasetuploadexception.webknossos.xyz

### Steps to test:
- attempt to upload dataset, should still work locally
- unless you remove write access to binaryData
- on dev servers, it seems, there is no write access to binaryData. That might be intentional? @normanrz 

### Issues:
- contributes to #2883

------
